PLATFORM-3333: Make all design system links protocol relative

themech opened the pull request.

Commit:
ADEN-6658 Adding audio=yes/no keyval to VAST

rybmat committed:
XW-4614: fandom change comment

Commit:
SUS-3933 Remove unused edit.php script

rybmat committed:
Merge branch 'dev' into XW-4614

Commit:
SUS-3934 Remove unused checkTableExists.php script

macbre committed:
SUS-4073 | remove dependency on globalimagelinks table on ext/wikia/FilePage

GlobalUsage extension was set up to use video151 when querying globalimagelinks. As we're sunseting video.wikia.com, this functionality is no longer needed.

Local, per-wiki file usage still works on file page.

Commit:
Merge branch 'dev' of github.com:Wikia/app into PLATFORM-3351

Commit:
SUS-3935 Remove unused testStore.php script

Grunny reviewed (changes requested):
Actually, we need to fix the unit tests which expect the protocol.

chessky committed:
Merge pull request #14691 from Wikia/SUS-3929

SUS-3929 Remove classifyNonArticleTemplates.class.php maintenance script

kvas-damian committed:
ADEN-6531 | Fixes after CR

Commit:
Merge remote-tracking branch 'origin/dev' into ADEN-6658

# Conflicts:
#	extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js

Commit:
SUS-3938 Remove unused dumpWikiFactorySettings.php script

Commit:
SUS-3937 Remove unused move_blobs.php script

macbre committed:
Merge pull request #14699 from Wikia/SUS-4073-remove-GlobalUsage-from-FilePage-ext

SUS-4073 | remove dependency on globalimagelinks table on ext/wikia/FilePage

macbre committed:
SUS-3897: drop global_registry

Commit:
Merge pull request #14616 from Wikia/PLATFORM-3351

PLATFORM-3351: https->http redirect exceptions for logged in users

chessky committed:
Merge pull request #14694 from Wikia/SUS-3930

SUS-3930 Remove unused checkArchiveFiles.php script

themech committed:
Merge pull request #14672 from Wikia/PLATFORM-3357

(PLATFORM-3357) Don't send dev CSP reports to prod service

rybmat committed:
Merge pull request #14697 from Wikia/XW-4614

XW-4614 | [RTE] check for transclusion of block templates inside span element

chessky committed:
Merge pull request #14686 from Wikia/SUS-3931

SUS-3931 Remove TemplateClassificationRecognizedMetric.php

Commit:
SUS-3939 Remove unused syncClusterWithCityList.php script

chessky committed:
Merge pull request #14695 from Wikia/SUS-3932

SUS-3932 Remove unused convert.php script

chessky committed:
Merge pull request #14696 from Wikia/SUS-3933

SUS-3933 Remove unused edit.php script

chessky committed:
Merge pull request #14698 from Wikia/SUS-3934

SUS-3934 Remove unused checkTableExists.php script

themech committed:
PLATFORM-3333: fixing design system unit tests

Grunny committed:
Merge branch 'dev' into allow-disable-downgrade

Commit:
SUS-3940 Remove unused clear_tag_stats.php script

themech committed:
PLATFORM-3333: don't use protocol-relative urls for license addresses

Grunny committed:
Merge pull request #14674 from Wikia/allow-disable-downgrade

Allow disabling HTTPS downgrade outside of dev and sandbox envs

Grunny committed:
Merge pull request #14342 from Wikia/PLATFORM-3303

(PLATFORM-3303) Unset default values for shared uploads

Commit:
SUS-3942 Remove unused syncSwiftImagesBetweenDC.php script

Commit:
SUS-3945 Remove unused removeRedundantUserNames.php script

Commit:
Automatic translations update

themech committed:
PLATFORM-3333: License url is protocol relative when generated from wgRightsPage

macbre committed:
SUS-4080 | WantedFilesPageWikia::reallyDoQuery - do not query video_info.premium column

Revert "VID-492 Ignore known premium video in the WantedFiles page"

This reverts commit 3bd4aa2d9677383a769f35a21228a90d903dd125.

macbre committed:
SUS-4080 | remove AddPremiumVideo hook and almost three years old TODO and commented out code

themech committed:
PLATFORM-3333: unit test for protocol-relative license urls

macbre committed:
VideoHandlers: remove handling of premium videos

macbre committed:
SUS-4080 | remove RemovePremiumVideo hook handling and remove premium videos support

macbre committed:
SUS-4080 | remove MediaQueryService::clearCacheTotalPremiumVideos

macbre committed:
SUS-4080 | remove MediaQueryService::getTotalPremiumVideos

macbre committed:
SUS-4080 | VideoInfo - remove addPremiumVideo and restorePremiumVideo methods

macbre committed:
SUS-4080 | VideoInfo - remove support for premium flag from video_info table

Commit:
SUS-3946 Remove unused deleteLocalGroup.php script

macbre committed:
SUS-4080 | VideoInfo::isPremium can go away + improve type hinting

Commit:
SUS-3948 Remove unused syncDevData.php script

Commit:
SUS-3949 Remove unused cleanupEditTags.php script

macbre committed:
Merge branch 'dev' into SUS-3618

macbre committed:
Merge pull request #14517 from Wikia/SUS-3618

SUS-3618 | update RenameUser i18n

rybmat committed:
XW-4639: prevent text from disappearing

themech committed:
PLATFORM-3351: Fix a redirect exception for non-main namespaces

rybmat committed:
XW-4639: simplify previous fixes

macbre committed:
SUS-4080 | remove first argument of MediaQueryService::getVideoList()

macbre committed:
Merge pull request #14709 from Wikia/SUS-4080

SUS-4080 | remove handling of video_info.premium column in app code

Commit:
SUS-3951 Remove unused imageServing.php script

Commit:
SUS-3953 Remove unused utf8/check.php script

Commit:
SUS-3954 Remove unused clearWikifactoryCache.php script

Commit:
SUS-3955 Remove unused deleteArticlesByTitle.php script

Commit:
SUS-3956 Remove unused utf8/blobtest.php script

Commit:
SUS-3957 Remove unused email.php script

Commit:
SUS-3958 Remove unused fixRedirects.php script

Commit:
SUS-3959 Remove unused asyncCache.php script

Commit:
SUS-3961 Remove unused synchronizeBlobs.php script

Commit:
SUS-3966 Remove unused pushUploadsToImageReview.php script

Commit:
SUS-3967 Remove unused ConfigureSmartBanner.php script

themech committed:
Merge pull request #14713 from Wikia/PLATFORM-3351-fix

PLATFORM-3351: Fix a redirect exception for non-main namespaces

Commit:
SUS-3969 Remove unused cleanupNewWikis.php script

Commit:
SUS-3971 Remove unused purgeArticlePages.php script

themech committed:
PLATFORM-3284: sunset discontinued Gametrailers provider

macbre committed:
Merge branch 'dev' into SUS-4072-WikiaVideoRepo-sunset

chessky committed:
Merge pull request #14687 from Wikia/SUS-4072-WikiaVideoRepo-sunset

SUS-4072 | cleanup after video.wikia.com sunset

Commit:
SUS-3972 Remove unused cleanPageWikiaProps.php script

chessky committed:
Merge pull request #14701 from Wikia/SUS-3935

SUS-3935 Remove unused testStore.php script

chessky committed:
Merge pull request #14703 from Wikia/SUS-3938

SUS-3938 Remove unused dumpWikiFactorySettings.php script

chessky committed:
Merge pull request #14711 from Wikia/SUS-3948

SUS-3948 Remove unused syncDevData.php script

chessky committed:
Merge pull request #14712 from Wikia/SUS-3949

SUS-3949 Remove unused cleanupEditTags.php script

chessky committed:
Merge pull request #14724 from Wikia/SUS-3966

SUS-3966 Remove unused pushUploadsToImageReview.php script

Commit:
SUS-3975 Remove unused updateIpblocksSchema.php script

Grunny committed:
(PLATFORM-3363) Fix using VisualEditor over HTTPS on production

rybmat committed:
XW-4639: use getText() instead getRawText() when quoting message on MessageWall

kvas-damian committed:
ADEN-6531 | Reuse articleVideoAd.buildVastUrl() in jwplayer tags video

macbre committed:
SUS-4083 | Remove support for Cortado Java applet from OggHandler extension

chessky committed:
Merge pull request #14705 from Wikia/SUS-3939

SUS-3939 Remove unused syncClusterWithCityList.php script

chessky committed:
Merge pull request #14710 from Wikia/SUS-3946

SUS-3946 Remove unused deleteLocalGroup.php script

macbre committed:
Remove ogg-player-cortado i18n string

macbre committed:
Merge pull request #14733 from Wikia/SUS-4083-remove-cortado-java-applet

SUS-4083 | Remove support for Cortado Java applet from OggHandler extension

Commit:
ADEN-6676 Inlining InstantGlobals

rybmat committed:
Merge pull request #14717 from Wikia/XW-4639

XW-4639 | [RTE] Improve handling <br /> and adding newlines to wikitext

Grunny committed:
Merge pull request #14732 from Wikia/PLATFORM-3363

(PLATFORM-3363) Fix using VisualEditor over HTTPS on production

Commit:
SUS-3977 Remove unused renameUser.php script

Commit:
SUS-3978 Remove unused RenameUserTaskManagerMaintenance.php script

fraszczakszymon committed:
ADEN-6610 Update ad-engine

fraszczakszymon committed:
ADEN-6610 Build ad-engine

fraszczakszymon committed:
Merge branch 'dev' into ADEN-6610

macbre committed:
SUS-3855 | Chat - track room joins and messages sent

Commit:
SUS-3980 Remove unused resendConfirmationEmail.php script

macbre committed:
Merge pull request #14738 from Wikia/SUS-3855-events-tracking-for-chat

SUS-3855 | Chat - track room joins and messages sent

macbre committed:
SUS-3855 | track private and public chat messages separately

rybmat committed:
XW-4626: apply EMPTY_LINES_BEFORE only to block elements

Commit:
SUS-3906 Remove unused MigrateReqCloseDateToPermService.php script

Commit:
SUS-3908 Remove unused moveScriptsToNamespace.php script

Commit:
SUS-3911 Remove unused CleanupCommentsIndex.php script

Commit:
SUS-3912 Remove unused setLastRevisionsAsReviewed.php script

fraszczakszymon committed:
ADEN-6666 Avoid decoding not encoded string

fraszczakszymon committed:
ADEN-6666 Add unit test

macbre committed:
Merge pull request #14740 from Wikia/SUS-3855-events-tracking-improved

SUS-3855 | track private and public chat messages separately

Commit:
SUS-3913 Remove unused FixImageLinks.php script

Commit:
SUS-3914 Remove unused test-solrdocumentservice-performance.php script

Commit:
SUS-3915 Remove unused setRevisionsBeforeDateAsReviewed.php script

fraszczakszymon committed:
ADEN-6610 Add setConfigProperty to slot interface

Commit:
SUS-3922 Remove unused migrateAvatarsViaService.php script

Commit:
SUS-3981 Remove unused moveMain.php script

Commit:
SUS-3983 Remove unused createNewWiki.php script

Commit:
SUS-3984 Remove unused AutoFollow.php script

Commit:
SUS-3987 Remove unused createUser.php script

Commit:
SUS-3985 Remove unused checktbl.php script

rybmat committed:
XW-4626: allow adding data-rte-empty-lines-before to tr

rybmat committed:
XW-4626: use div for block template placeholder even if markup inside is incorrect and puzzle will be displayed

Commit:
SUS-3974 Remove unused utils.php script

Commit:
SUS-3982 Remove unused purgeImages.php script

Commit:
SUS-3920 Remove unused reindex-wiki.php script

kvas-damian committed:
ADEN-6584 | Desktop BFAA unstick animation

Commit:
ADEN-6700 Porvata mute/unmute events tracking

fraszczakszymon committed:
ADEN-6610 Remove console log

Commit:
ADEN-6676 CR fixes

chessky committed:
Merge pull request #14726 from Wikia/SUS-3969

SUS-3969 Remove unused cleanupNewWikis.php script

chessky committed:
Merge pull request #14707 from Wikia/SUS-3942

SUS-3942 Remove unused syncSwiftImagesBetweenDC.php script

chessky committed:
Merge pull request #14708 from Wikia/SUS-3945

SUS-3945 Remove unused removeRedundantUserNames.php script

chessky committed:
Merge pull request #14716 from Wikia/SUS-3954

SUS-3954 Remove unused clearWikifactoryCache.php script

chessky committed:
Merge pull request #14725 from Wikia/SUS-3967

SUS-3967 Remove unused ConfigureSmartBanner.php script

Commit:
Automatic translations update

Commit:
ADEN-6676 More CR fixes

macbre committed:
PartnerFeed: fix a misleading comment

rybmat committed:
Merge pull request #14758 from Wikia/XW-4626

XW-4626 | [RTE] Fix line-breaks

kvas-damian committed:
ADEN-6584 | Separate unstick callback

kvas-damian committed:
ADEN-6584 | BFAA animation mobile

Commit:
ADEN-6676 Change default TTL variable

bulaszenko committed:
Merge pull request #14684 from Wikia/ADEN-6610

ADEN-6610 Add audio flag to ad requests

kvas-damian committed:
ADEN-6584 | Remove unused import

kvas-damian committed:
ADEN-6584 | Fix function name

bulaszenko committed:
Merge pull request #14702 from Wikia/ADEN-6658

ADEN-6658 Adding audio=yes/no keyval to VAST

macbre committed:
SUS-3268 | remove PowerUserTrait and site-wide include for PowerUser extension

macbre committed:
SUS-3268 | git rm -r extensions/wikia/PowerUser

macbre committed:
SUS-3268 | Remove poweruser from PermissionsConfigurationImpl

macbre committed:
SUS-3268 | Remove poweruser from AnalyticsEngine/js/universal_analytics.js

macbre committed:
SUS-3268 | Remove poweruser from SiteWideMessages

macbre committed:
SUS-3268 | Remove poweruser from Qualaroo

macbre committed:
SUS-3268 | Remove poweruser from UserApiController

macbre committed:
SUS-3268 | Remove poweruser from MercuryApiController

macbre committed:
SUS-3268 | Remove poweruser from ListusersData

Reverts 1c8c182cab8e8e62e0768bcf459a3964e70ab43e

macbre committed:
SUS-3268 | Remove poweruser from SpecialListusers

Revert "SUS-1478: Exclude powerusers from listusers group list"

This reverts commit 3aaa2756bd6c7d16150394743315453c27e514b1.

macbre committed:
SUS-3268 | the ultimate app code cleanup

kvas-damian committed:
ADEN-6584 | Use ad-products@4.0.0

macbre committed:
Merge pull request #14763 from Wikia/SUS-3268-sunset-poweruser

SUS-3268 | sunset PowerUser feature

kvas-damian committed:
ADE-6584 | Fix mobile Hivi BFAA

Commit:
SUS-4084: Optimize Forum Activity module query

kvas-damian committed:
Merge remote-tracking branch 'origin/dev' into ADEN-6584

fraszczakszymon committed:
ADEN-6696 Add possibility to cache predictions

fraszczakszymon committed:
ADEN-6696 Calculate models predictions once

fraszczakszymon committed:
ADEN-6696 Refactor rabbit model so we can get needed results

fraszczakszymon committed:
ADEN-6696 Add instant global

fraszczakszymon committed:
ADEN-6696 Add rabbit key-val

fraszczakszymon committed:
Merge pull request #14734 from Wikia/ADEN-6676

ADEN-6676 Inlining InstantGlobals

rybmat committed:
XW-4610: update explore wikis for Feb

macbre committed:
SUS-4005 | Wikia's external user is gone, no need to keep the core support for it

"external_user" per-wiki table has been removed in MediaWiki 1.22 along with the ExternalAuth functionality. AuthPlugin should be used instead.

macbre committed:
SUS-4005 | stop creating per-wiki external_user table

JamesHawking committed:
Add enableDiscussionsPolls to MercuryApi

bulaszenko committed:
Merge pull request #14746 from Wikia/ADEN-6666

ADEN-6666 Avoid decoding not encoded string

macbre committed:
Merge pull request #14767 from Wikia/SUS-4005-remove-ExternalUser-core-support

SUS-4005 | Wikia's external user is gone, no need to keep the core support for it

Commit:
SUS-4084: Correctly handle multiple reply revisions

Commit:
Automatic translations update

fraszczakszymon committed:
ADEN-6696 Refactor filtering models

fraszczakszymon committed:
ADEN-6696 Reset prediction on page change

fraszczakszymon committed:
ADEN-6696 Add unit tests

kvas-damian committed:
ADEN-6584 | Fix global nav position on fast scroll up

fraszczakszymon committed:
ADEN-6696 Extract method

Commit:
Merge pull request #14647 from Wikia/PLATFORM-3292

(PLATFORM-3292) Support HTTPS in GlobalTitle

latata committed:
Merge pull request #14765 from Wikia/XW-4610

XW-4610 | update explore wikis for Feb

chessky committed:
Merge pull request #14757 from Wikia/SUS-3982

SUS-3982 Remove unused purgeImages.php script

chessky committed:
Merge pull request #14720 from Wikia/SUS-3957

SUS-3957 Remove unused email.php script

kvas-damian committed:
ADEN-6546 | Fix Banner notification position with UAP:HiVi

kvas-damian committed:
ADEN-6584 | Fix global nav on scroll one more time

Commit:
Fix for unavailable GlobalShortcutsSearch module

kvas-damian committed:
ADEN-6584 | Fix global nav one more time

latata committed:
Merge pull request #14774 from Wikia/fix-globalshortcutsearch

Fix for unavailable GlobalShortcutsSearch module

tortila committed:
Merge pull request #14768 from Wikia/IRIS-5243

IRIS-5243 | Poll feature flag

kvas-damian committed:
ADEN-6723 | Fix tracking

kvas-damian committed:
ADEN-6584 | Revert global nav position fix

macbre committed:
SUS-4165 | remove wgUseVideoVerticalFilters handling

macbre committed:
Merge pull request #14777 from Wikia/SUS-4165-remove-wgUseVideoVerticalFilters-handling

SUS-4165 | remove wgUseVideoVerticalFilters handling

fraszczakszymon committed:
ADEN-6696 Fix empty instant global

bulaszenko committed:
Merge pull request #14773 from Wikia/ADEN-6546

ADEN-6546 | Fix Banner notification position with UAP:HiVi

bulaszenko committed:
Merge pull request #14664 from Wikia/ADEN-6584

ADEN-6584 Adds animation to HiVi BFAA

themech committed:
PLATFORM-3364: adjust csp header

kvas-damian committed:
Merge pull request #14775 from Wikia/ADEN-6723

ADEN-6723 | Fix tracking

chessky committed:
Merge pull request #14770 from Wikia/SUS-4084

SUS-4084: Optimize Forum Activity module query

Commit:
ADEN-6531 merge dev and resolve conflicts

Commit:
Automatic translations update

Commit:
Merge pull request #14650 from Wikia/ADEN-6531

ADEN-6531 Disable jwplayer tag ads when UAP is loaded

macbre committed:
SUS-3279 | SunsetProvider - fallback to image table, video_info is out of sync on mediawiki119.wikia.com

macbre committed:
SunsetProvider | add a line-break

macbre committed:
Merge pull request #14782 from Wikia/SUS-3279-sunset-provider-image-table-fallback

SUS-3279 | SunsetProvider - fallback to image table, video_info is out of sync on mediawiki119.wikia.com

themech committed:
Merge pull request #14779 from Wikia/PLATFORM-3364-crossorigin-cleanup

PLATFORM-3364: adjust csp header

fraszczakszymon committed:
ADEN-6696 Don't send empty key-val

macbre committed:
SUS-4074 | sunset extensions/wikia/VideosModule

This "removed" in January 2016 as a part of an experiment with Taboola. Now do it fully :)

See https://github.com/Wikia/app/commit/e91d6f92e088d71043a5c2c2d01c6c2ec55a2d98#diff-40462e8426349e2fb2bd4a435c1d8473

macbre committed:
SUS-4074 | VideosModule assets cleanup

ludwikkazmierczak committed:
Merge pull request #14728 from Wikia/PLATFORM-3284

PLATFORM-3284: sunset discontinued Gametrailers provider

bulaszenko committed:
Merge pull request #14764 from Wikia/ADEN-6696

ADEN-6696 Add rabbit results to targeting key-vals

kolek7 committed:
XW-4648 | Using getRawText() instead of getText()

Commit:
Merge pull request #14759 from Wikia/SUS-3920

SUS-3920 Remove unused reindex-wiki.php script

Commit:
Merge pull request #14755 from Wikia/SUS-3985

SUS-3985 Remove unused checktbl.php script

Commit:
Merge pull request #14715 from Wikia/SUS-3953

SUS-3953 Remove unused utf8/check.php script

Commit:
Merge pull request #14756 from Wikia/SUS-3974

SUS-3974 Remove unused utils.php script

Commit:
Merge pull request #14754 from Wikia/SUS-3987

SUS-3987 Remove unused createUser.php script

Commit:
Merge pull request #14753 from Wikia/SUS-3984

SUS-3984 Remove unused AutoFollow.php script

Commit:
Merge pull request #14752 from Wikia/SUS-3983

SUS-3983 Remove unused createNewWiki.php script

Commit:
Merge pull request #14751 from Wikia/SUS-3981

SUS-3981 Remove unused moveMain.php script

Commit:
Merge pull request #14750 from Wikia/SUS-3922

SUS-3922 Remove unused migrateAvatarsViaService.php script

Commit:
Merge pull request #14749 from Wikia/SUS-3915

SUS-3915 Remove unused setRevisionsBeforeDateAsReviewed.php script

Commit:
Merge pull request #14748 from Wikia/SUS-3914

SUS-3914 Remove unused test-solrdocumentservice-performance.php script

Commit:
Merge pull request #14747 from Wikia/SUS-3913

SUS-3913 Remove unused FixImageLinks.php script

Commit:
Merge pull request #14745 from Wikia/SUS-3912

SUS-3912 Remove unused setLastRevisionsAsReviewed.php script

Commit:
Merge pull request #14744 from Wikia/SUS-3911

SUS-3911 Remove unused CleanupCommentsIndex.php script

Commit:
Merge pull request #14743 from Wikia/SUS-3908

SUS-3908 Remove unused moveScriptsToNamespace.php script

Commit:
Merge pull request #14742 from Wikia/SUS-3906

SUS-3906 Remove unused MigrateReqCloseDateToPermService.php script

Commit:
Merge pull request #14739 from Wikia/SUS-3980

SUS-3980 Remove unused resendConfirmationEmail.php script

Commit:
Merge pull request #14737 from Wikia/SUS-3978

SUS-3978 Remove unused RenameUserTaskManagerMaintenance.php script

Commit:
Merge pull request #14735 from Wikia/SUS-3977

SUS-3977 Remove unused renameUser.php script

Commit:
Merge pull request #14730 from Wikia/SUS-3975

SUS-3975 Remove unused updateIpblocksSchema.php script

Commit:
Merge pull request #14729 from Wikia/SUS-3972

SUS-3972 Remove unused cleanPageWikiaProps.php script

Commit:
Merge pull request #14727 from Wikia/SUS-3971

SUS-3971 Remove unused purgeArticlePages.php script

Commit:
Merge pull request #14723 from Wikia/SUS-3961

SUS-3961 Remove unused synchronizeBlobs.php script

Commit:
Merge pull request #14722 from Wikia/SUS-3959

SUS-3959 Remove unused asyncCache.php script

Commit:
Merge pull request #14721 from Wikia/SUS-3958

SUS-3958 Remove unused fixRedirects.php script

Commit:
Merge pull request #14719 from Wikia/SUS-3956

SUS-3956 Remove unused utf8/blobtest.php script

Commit:
Merge pull request #14718 from Wikia/SUS-3955

SUS-3955 Remove unused deleteArticlesByTitle.php script

Commit:
Merge pull request #14714 from Wikia/SUS-3951

SUS-3951 Remove unused imageServing.php script

Commit:
Merge pull request #14706 from Wikia/SUS-3940

SUS-3940 Remove unused clear_tag_stats.php script

Commit:
Merge pull request #14704 from Wikia/SUS-3937

SUS-3937 Remove unused move_blobs.php script

dmnsobczak committed:
Merge pull request #14669 from Wikia/ADEN-6289

ADEN-6289 Clean up variables

bulaszenko committed:
Merge pull request #14761 from Wikia/ADEN-6700

ADEN-6700 Porvata mute/unmute events tracking

Commit:
ADEN-6700 Bump ad-engine and ad-products versions

dmnsobczak committed:
Merge pull request #14791 from Wikia/ADEN-6700-2

ADEN-6700 Bump ad-engine and ad-products versions

chessky committed:
Merge pull request #14784 from Wikia/SUS-4074-sunset-videos-module-extension

SUS-4074 | sunset extensions/wikia/VideosModule  …

Commit:
Merge pull request #14786 from Wikia/XW-4648

XW-4648 | Quoting posts breaks wikitext code in the quoted post.

Commit:
Automatic translations update

kvas-damian committed:
ADEN-6735 | Fix palyMuted attribute in buildVastUrl

kvas-damian committed:
ADEN-6735 | Get audio back to MEGA vast

fraszczakszymon committed:
Merge pull request #14794 from Wikia/ADEN-6735

ADEN-6735 | Fix palyMuted attribute in buildVastUrl

Commit:
Merge branch 'dev' into PLATFORM-3333

ludwikkazmierczak commented:
Moved to : https://github.com/Wikia/app/pull/14797
